autoscale/src/main/java/com/sequenceiq/periscope/service/ScalingActivityService.java

```diff
@@ -110,7 +110,7 @@ public void save(ScalingActivity scalingActivity) {
         scalingActivityRepository.save(scalingActivity);
     }
 
-    private String createScalingActivityCrn(String accountId) {
+    private String createScalingActivityId(String accountId) {
         return regionAwareCrnGenerator.generateCrnStringWithUuid(CrnResourceDescriptor.SCALING_ACTIVITY, accountId);
     }
```

Comment:
This pathc essentially changes nothing because it's still producing a CRN. We will have to remove it from the Crn class in CB and thundehread

Reply:
Changed it to UUID.randomUUID().toString()